src/Components/ContrastChecker/ContrastChecker.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from "react";
+import "./ContrastChecker.css";
+
+export default function ContrastChecker({ firstColor, secondColor }) {
+  const [contrastResult, setContrastResult] = useState(null);
+
+  useEffect(
+    (contrastResult) => {
```

Comment:
I think the useEffect callback function should not take parameters because the dependency array ensures that the effect is re-run whenever the dependencies change.

Reply:
Thank you. First I had not passed the usestate constant to the use effect as an argument, but then the dependency array was underlined and the error message was:

React Hook useEffect has a missing dependency: 'contrastResult'. Either include it or remove the dependency array.

:woman_shrugging:

Reply:
Aaaaah, you know what? The error message only came up because of the console.logs, I think. And I was wondering why there was a problem there. :rofl: :zany_face:

Reply:
Try to Include contrastResult in the dependency array..